### ISSUE TITLE
User Images in Gallery

### ISSUE THREAD

**Issue by woodytexas** [modrinth/knossos#425]
### Is your feature request related to a problem? Please describe.

Currently, only the project team can upload gallery images depending on permissions.

### Describe the solution you'd like

Allow for users to upload images to a mod's gallery. Also have an option for project teams to turn off user-uploaded images for their galleries.

**Comment by wafflecoffee** [comment]
This sounds like a massive recipe for disaster, with "griefing" of gallery pages...

**Comment by woodytexas** [comment]
so its better off that only project teams can add gallery images?

**Comment by enderger** [comment]
Perhaps it should be opt-in rather than opt-out.

**Comment by Minenash** [comment]
If this were added, I'd agree opt-in, but even then, this sounds like a bad idea unless it was more of a "submit picture" with the mod author approving/denying them, kinda like how file moderators approve mods.

But this seems like a lot of work to be an alternative to contacting the mod author and giving the img to them directly

**Comment by MulverineX** [comment]
Yeah and it could be abused as an alternative to comment sections, which Modrinth has already decided to not do.

**Comment by MulverineX** [comment]
Well a big part of the comments thing is harassment, and those types of people will look for an opportunity.

**Comment by PssbleTrngle** [comment]
> Well a big part of the comments thing is harassment, and those types of people will look for an opportunity.

You are correct,

**Comment by Arbee4ever** [comment]
Why not make the mods team to approve these images and set who is allowed to add images.

**Comment by wafflecoffee** [comment]
@Arbee4ever That's a pretty massive moderation burden. If implemented, this would end up having to be approved by the project team members rather than by moderators.

**Comment by Geometrically** [comment]
We will add comments eventually, it's not true that we have decided to not have them

**Comment by Arbee4ever** [comment]
> @Arbee4ever That's a pretty massive moderation burden. If implemented, this would end up having to be approved by the project team members rather than by moderators.

I meant the project team members with mods team. I just forgot the ' in mods (like this: "mod's") Sorry for that misunderstanding :D

**Comment by MulverineX** [comment]
re: comments
Oh apologies I seemed to have misunderstood or had out of date information.
